src/main/scala/cz/alenkacz/db/postgresscala/Implicits.scala

```diff
@@ -0,0 +1,26 @@
+package cz.alenkacz.db.postgresscala
+
+object Implicits {
```

Comment:
I would prefer to have this implicit class in `package.scala` so `import cz.alenkacz.db.postgresscala._` would be required (no `Implicits` there).